src/pages/monitor/backup-and-restore.html

```diff
@@ -0,0 +1,74 @@
+<div>
+    <title>${messages.app.title} : ${messages.jmx.QueryMonitoringPage.title}</title>
+
+    <link href="css/backup-and-restore.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+    <h1>
+        {{title}}
+        <span class="btn btn-link"
+              uib-popover-template="'js/angular/templates/titlePopoverTemplate.html'"
+              popover-trigger="mouseenter"
+              popover-placement="bottom-right"
+              popover-append-to-body="true">
+            <span class="icon-info text-tertiary"></span>
+        </span>
+        <span onto-loader-new class="backup-and-restore-updater-icon" size="25">
+        </span>
+    </h1>
+
+    <div core-errors ontop></div>
+
+    <div ng-if="initialized" class="backup-and-restore">
+        <table ng-if="backupAndRestoreInfos && backupAndRestoreInfos.length > 0" class="table table-striped table-bordered"
+               aria-describedby="Monitor backup and restore">
+            <thead>
+            <tr>
+                <th class="id-header">{{'view.monitoring.backup_and_restore.id.header' | translate}}</th>
+                <th class="username-header">{{'view.monitoring.backup_and_restore.username.header' | translate}}</th>
+                <th class="recovery-operation-header">{{'view.monitoring.backup_and_restore.recovery_operation_type.header' | translate}}
+                </th>
+                <th class="affected-repository-header">{{'view.monitoring.backup_and_restore.affected_repositories.header' | translate}}
+                </th>
+                <th class="lifetime-header">{{'view.monitoring.backup_and_restore.lifetime.header' | translate}}</th>
+                <th class="snapshot-options-header">{{'view.monitoring.backup_and_restore.snapshot_options.header' | translate}}</th>
+                <th ng-if="hasClusterOperation" class="node-performing-cluster-backup-header">
```

Comment:
Maybe we should always display the columns regardless if they have info. If some operation starts it will bounce the table.

Reply:
The column contains information only for operations executed on the cluster. Currently, we support only one backup or restore operation at a time, resulting in a single type of operation that either shows or does not show this column. After discussing it with the team, we decided to display it only when necessary.

Reply:
So this could not spawn while you are on the page without doing anything in this view?

Reply:
Yes.